<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar señales a un .py - MEG Software</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="btAbrirPy">
          <property name="toolTip">
           <string>Abrir archivo Python</string>
          </property>
          <property name="text">
           <string>Archivo Python</string>
          </property>
          <property name="icon">
           <iconset resource="images/qtgui_art.qrc">
            <normaloff>:/images/python-file.png</normaloff>:/images/python-file.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btAbrirUI">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Archivo UI</string>
          </property>
          <property name="icon">
           <iconset resource="images/qtgui_art.qrc">
            <normaloff>:/images/ui-file.png</normaloff>:/images/ui-file.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Archivo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbArchivos">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Widgets</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QTreeView" name="treeWidgets">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="autoScroll">
             <bool>false</bool>
            </property>
            <property name="autoScrollMargin">
             <number>16</number>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="defaultDropAction">
             <enum>Qt::MoveAction</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Señales</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabPosition">
             <enum>QTabWidget::East</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Triangular</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <property name="usesScrollButtons">
             <bool>true</bool>
            </property>
            <property name="documentMode">
             <bool>true</bool>
            </property>
            <property name="tabsClosable">
             <bool>false</bool>
            </property>
            <property name="movable">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Agregar</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QListWidget" name="lwSenales">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QPushButton" name="btAgregarSenial">
                  <property name="maximumSize">
                   <size>
                    <width>28</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="images/qtgui_art.qrc">
                    <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Quitar</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QCheckBox" name="chkAgregarSelf">
                <property name="text">
                 <string>Agregar &lt; self. &gt;</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QTextEdit" name="teWidgets">
                  <property name="verticalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOn</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QPushButton" name="btQuitarSenial">
                    <property name="maximumSize">
                     <size>
                      <width>28</width>
                      <height>24</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="images/qtgui_art.qrc">
                      <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>200</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSaveFile">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Guardar las modificaciones en el archivo.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images/qtgui_art.qrc">
            <normaloff>:/images/xfce-mount.png</normaloff>:/images/xfce-mount.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btUndo">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Deshacer el cambio inmediatamenten anterior.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images/qtgui_art.qrc">
            <normaloff>:/images/gtk-jump-to-rtl.png</normaloff>:/images/gtk-jump-to-rtl.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Archivo fuente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QsciScintilla" name="qscArchivo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>350</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btAddToFile">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Insertar en el archivo en la posicion seleccionada.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images/qtgui_art.qrc">
            <normaloff>:/images/rew.png</normaloff>:/images/rew.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSaveSignals">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Guardar este código en un fuente .py</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images/qtgui_art.qrc">
            <normaloff>:/images/xfce-mount.png</normaloff>:/images/xfce-mount.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btCopyCliptboard">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Copiar al portapapeles</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images/qtgui_art.qrc">
            <normaloff>:/images/clipt.png</normaloff>:/images/clipt.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btAddIdentation">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Agregar identacion de 4 espacios.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images/qtgui_art.qrc">
            <normaloff>:/images/identation.png</normaloff>:/images/identation.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Metodos de las señales generadas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QsciScintilla" name="qscSignals">
          <property name="toolTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/qtgui_art.qrc"/>
 </resources>
 <connections/>
</ui>
